docs/changelog-fragments.d/526.feature.md

```diff
@@ -0,0 +1,2 @@
+Add ability to run commands directly within an execution environment.
```

Comment:
Per https://github.com/ansible/ansible-navigator/pull/641#discussion_r765392465, something like this may be better:
```suggestion
Added the ability to run arbitrary shell commands from the TUI directly
so they are invoked within an execution environment.
```